west.yml

```diff
@@ -28,6 +28,9 @@ manifest:
     - name: canopennode
       revision: 53d3415c14d60f8f4bfca54bfbc5d5a667d7e724
       path: modules/lib/canopennode
+    - name: chre
+      revision: 2e1ba542aeea185d801830490e20fc93f436912f
```

Comment:
> Need to use https://github.com/zephyrproject-rtos/chre/pull/1 to get rid of the warnings

```suggestion
      revision: pull/1/head
```

Reply:
This change indeed eliminates the warnings